btcutil/amount_test.go

```diff
@@ -136,8 +136,22 @@ func TestAmountUnitConversions(t *testing.T) {
 			name:      "BTC",
 			amount:    44433322211100,
 			unit:      AmountBTC,
-			converted: 444333.22211100,
-			s:         "444333.222111 BTC",
+			converted: 444333.222111,
+			s:         "444333.22211100 BTC",
+		},
+		{
+			name:      "a thousand satoshi as BTC",
+			amount:    1000,
+			unit:      AmountBTC,
+			converted: 0.00001,
+			s:         "0.00001000 BTC",
+		},
+		{
+			name:      "a single satoshi as BTC",
+			amount:    1,
+			unit:      AmountBTC,
+			converted: 0.00000001,
+			s:         "0.00000001 BTC",
```

Comment:
add a testcase for BTC without decimals (like 10.0 BTC)

Reply:
Added